scvi/model/base/_base_model.py

```diff
@@ -40,21 +40,33 @@ class BaseModelMetaClass(ABCMeta):
     """
     Metaclass for :class:`~scvi.model.base.BaseModelClass`.
 
-    Constructs a model class-specific mapping for :class:`~scvi.data.anndata.AnnDataManager` instances.
+    Constructs model class-specific mappings for :class:`~scvi.data.anndata.AnnDataManager` instances.
+    ``cls._setup_adata_manager_store`` maps from AnnData object UUIDs to :class:`~scvi.data.anndata.AnnDataManager` instances.
+    This mapping is populated everytime ``cls.setup_anndata()`` is called.
+    ``cls._per_isntance_manager_store`` maps from model instance UUIDs to AnnData UUID::class:`~scvi.data.anndata.AnnDataManager` mappings.
+    These :class:`~scvi.data.anndata.AnnDataManager` instances are tied to a single model instance and populated either
+    during model initialization or after running ``self._validate_anndata()``.
     """
 
     def __init__(cls, name, bases, dct):
-        cls.manager_store = dict()
+        cls._setup_adata_manager_store: Dict[
```

Comment:
correct me if I'm wrong, but this manager store will only ever receive one anndata now, the one used to initialize the model?

Reply:
if this is the case, should we make it not a dict?

Reply:
actually this is general enough...

Reply:
it stores it for the whole class. this allows for situations where an AnnData is setup once and used to create two separate models.